src/main/java/eggventory/Parser.java

```diff
@@ -72,20 +97,20 @@ private Command handleListInput(String listInput) throws BadInputException,
             command = new DoneCommand(CommandType.DONE, Integer.parseInt(inputArr[1]));
             break;
         case "delete":
+            inputArr[1] = inputArr[1].strip();
             command = new DeleteCommand(CommandType.DELETE, Integer.parseInt(inputArr[1]));
             break;
 
         //Commands which require string input.
```

Comment:
To remove this comment, make the code more descriptive

Reply:
ok